session/session_test/BUILD.bazel

```diff
@@ -0,0 +1,26 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_test")
+
+go_test(
+    name = "session_test_test",
```

Comment:
Maybe you can make a dedicated name. package session_test is ambitious.

Reply:
OK, I agree with you.